core-s3/src/test/scala/io/aiven/guardian/kafka/s3/S3Spec.scala

```diff
@@ -50,7 +50,7 @@ trait S3Spec
       )
 
   implicit val ec: ExecutionContext            = system.dispatcher
-  implicit val defaultPatience: PatienceConfig = PatienceConfig(10 minutes, 100 millis)
+  implicit val defaultPatience: PatienceConfig = PatienceConfig(20 minutes, 100 millis)
```

Comment:
Just a remark: seems like with 20 mins timeout you could poll significantly less frequently, 100 milis -> 10 seconds?

Reply:
The current problem is that there are quite a few `Future` based tests (its Scala's `Future` that uses the `PatienceConfig` which run quite quickly and and then the rest are tests that run for a long time. There is an argument to apply a specific `PatienceConfig` depending on the test but this is overkill for now.